### ISSUE TITLE
Volume Bug - SWAMP Room Startup Volume

### ISSUE THREAD

**Issue by DR7** [adelyte/crescendo#39]
When the SWAMP Room(s) is set for example with a startup room volume of 30% and a user turns on the room and increases volume of the source to lets say 42% and then powers OFF the room.

On the next startup of the room the volume on the template will show the volume is at 30% however with a single press of the Volume UP button the volume will jump to 43%

I believe this happens as you are saving the room volume in ARAM and it restores the value when the room is powered back on. 

Perhaps a better solution would be for the Room Startup Volume to be included as a parameter in the "Room Module" and then a users would NOT set a startup value in the SWAMP. Optionally the user can choose a "Room Startup Volume" OR "Use Last Room Volume"

An example of how the current situation can cause problems is when the night before a person in in the Ensuite having a bath/shower and has the music at 50% volume and the next morning turns on the system and its at 30% as usual and he touches the volume and it jumps to 51% instantly...at 5:00 a.m its quite the shock ;-)

Thanks

**Comment by adelyte-austin** [comment]
Thank you for reporting this issue DR7. A simple solution will be to add an `abuf` to sync Volume_Is & Volume_Set when the zone powers on. Without using `aram` logic like you suggested, the programmed settings and those done in Sonnex Tool tend to compete with one another.

If you would like your program to manage startup levels in SIMPL rather than Toolbox, I recommend adding an `init` in the main program to set the values, using the 'Z__System_Initialized!'. Combined with the bug fix, that should address the two issues you're facing.

**Comment by DR7** [comment]
Thank you Austin, I will implement this fix today.

**Comment by adelyte-austin** [comment]
Great. Please report back with the results.

**Comment by DR7** [comment]
Austin, while testing I found if I use 'Z__System_Initialized!' this did not work proper and I had the same problem as before, I looked in debugger and noticed this signal is low while the program is running. 

I tried running it with a "1" instead and it works the way I want and basically I have an abuf with X-xx__Volume Is coming in and X-01-Volume going out of the abuf and it seems to work just how I want it now.

**Note I also removed the settings from the SWAMP to start the zone(s) at 30% and instead I am using an init to pulse X-xx_Startup_Volume.
